presentation/src/main/java/dk/nodes/template/presentation/ui/splash/SplashFragment.kt

```diff
@@ -0,0 +1,99 @@
+package dk.nodes.template.presentation.ui.splash
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AlertDialog
+import dk.nodes.nstack.kotlin.NStack
+import dk.nodes.nstack.kotlin.models.AppUpdate
+import dk.nodes.nstack.kotlin.models.AppUpdateState
+import dk.nodes.nstack.kotlin.models.Message
+import dk.nodes.nstack.kotlin.models.RateReminder
+import dk.nodes.template.presentation.R
+import dk.nodes.template.presentation.extensions.observeNonNull
+import dk.nodes.template.presentation.nstack.Translation
+import dk.nodes.template.presentation.ui.base.BaseFragment
+import dk.nodes.template.presentation.ui.main.*
+import dk.nodes.template.presentation.ui.sample.SampleViewModel
+
+class SplashFragment : BaseFragment() {
+
+    private val viewModel by viewModel<SplashViewModel>()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        viewModel.viewState.observeNonNull(this) { state ->
```

Comment:
This can result on multiple subscribes because of Fragment lifecycle, do either `viewModel.viewState.observeNonNull(this)` at `onActivitiyCreated` or `viewModel.viewState.observeNonNull(viewLifecycleOwner)` at `onViewCreated`